spectator-agent/src/main/resources/bitbucket.conf

```diff
@@ -5,126 +5,285 @@ netflix.spectator.agent.jmx {
       query = "com.atlassian.bitbucket.thread-pools:name=*"
       measurements = [
         {
-          name = "bitbucket.{name}.MaximumPoolSize"
+          name = "bitbucket.threadpool.maximumPoolSize"
           value = "{MaximumPoolSize}"
-          tags = [ ]
+          tags = [
+            {
+              key = "type"
```

Comment:
Due to some unfortunate historical baggage with `type` I would recommend avoiding it. In particular servo will automatically overwrite that tag. I don't think it will matter for this specific use-case, but just to avoid any problems. Many things use `id` as the generic key for this sort of thing, e.g.:

http://netflix.github.io/spectator/en/latest/ext/jvm-memory-pools/#metric-dimensions